By default the Makefile ignores the LDFLAGS specified in
/etc/abuild.conf make sure these are respected.

diff -upr opendetex-2.8.5.orig/Makefile opendetex-2.8.5/Makefile
--- opendetex-2.8.5.orig/Makefile	2019-08-18 13:47:35.008004838 +0200
+++ opendetex-2.8.5/Makefile	2019-08-18 13:48:24.344354667 +0200
@@ -110,7 +110,7 @@ VERSION = 2.8.5
 all:	${PROGS}
 
 detex: ${D_OBJ}
-	${CC} ${CFLAGS} -o $@ ${D_OBJ} ${LEXLIB}
+	${CC} ${CFLAGS} -o $@ ${D_OBJ} ${LEXLIB} ${LDFLAGS}
 
 delatex: detex
 	cp detex delatex
